ui/src/templates/api/index.ts

```diff
@@ -97,14 +110,17 @@ const createLabelsFromTemplate = async <T extends TemplateBase>(
   )
 
   const createdLabels = await client.labels.createAll(labelsToCreate)
+  const allLabels = [...createdLabels, ...existingLabels]
+
+  const labelMap: LabelMap = {}
 
-  // IDs of newly created labels that should be added to dashboard
-  const createdLabelIDs = createdLabels.map(l => l.id || '')
+  includedLabels.forEach(label => {
```

Comment:
is there a reason for using a foreach rather than a reduce?

Reply:
Juggling the accumulator and having to return the `labelMap` on each iteration when it just key-val update seemed like more overhead.